plugins/SitesManager/lang/en.json

```diff
@@ -67,7 +67,8 @@
         "SiteSearchUse": "You can use Piwik to track and report what visitors are searching in your website's internal search engine.",
         "SiteWithoutDataTitle": "No data has been recorded yet",
         "SiteWithoutDataDescription": "No analytics data has been tracked for this website yet.",
-        "SiteWithoutDataSetupTracking": "Please set up the %1$sJavaScript tracking code%2$s on your website and refresh the page.",
+        "SiteWithoutDataSetupTracking": "Please set up the %1$sJavaScript tracking code%2$s on your website if you haven't done that already.",
+        "SiteWithoutDataSetupGoals": "This message will disappear as soon as some data was tracked for this website. Meanwhile you may already start with setting up %1$ssome goals%2$s.",
```

Comment:
Maybe also split the sentence into 2 strings so we can edit second one without changing the first :) 

`Meanwhile you may already start with setting up %1$ssome goals%2$s.` -> `In the meantime, maybe you would like to %1$sset up some Goals%2$s?` or so?